cinema/models.py

```diff
@@ -85,9 +81,7 @@ def clean(self):
             (self.row, "row", "count_rows"),
             (self.seat, "seat", "count_seats_in_row"),
         ]:
-            count_attrs = getattr(
-                self.movie_session.cinema_hall, cinema_hall_attr_name
-            )
+            count_attrs = getattr(self.movie_session.cinema_hall, cinema_hall_attr_name)
```

Comment:
The attribute names 'count_rows' and 'count_seats_in_row' used in the `clean` method do not match the actual attribute names in the `CinemaHall` model, which are 'rows' and 'seats_in_row'. This will cause an `AttributeError`. Update the attribute names to 'rows' and 'seats_in_row' to match the `CinemaHall` model.